src/app/static/src/app/components/Stepper.vue

```diff
@@ -157,7 +157,7 @@
         },
         watch: {
             complete: function (){
-                if (this.activeStep === 4 && this.isComplete(4) && this.isEnabled(5)){
```

Comment:
This change was to prevent a potential bug in the code which I hadn't considered. The function was being triggered whenever complete state changed in either direction from true to false. This meant that when step 4 was made incomplete (and step 5 was still enabled), the stepper would advance, which is not ideal. The above code made sure that the watch function only cared about the complete state going from false to true and not true to false.

Reply:
Ah ok, will reinstate